daffodil-runtime1/src/main/scala/org/apache/daffodil/dpath/DFDLXFunctions.scala

```diff
@@ -81,20 +81,18 @@ case class DFDLXRightShift(recipes: List[CompiledDPath], argType: NodeInfo.Kind)
     val width = argType.asInstanceOf[PrimNumeric].width.get
     Assert.invariant(shift >= 0)
     if(shift>=width)
-      dstate.SDE("Right shift not supported for shift greater or equal to "+ width)
+      dstate.SDE("Left shift not supported for shift greater or equal to %s for %s. Shift was %s", width, argType.globalQName, shift)
```

Comment:
Copy/paste error. This should say Right shift. Now that I think about it, a better diagnostic would also say 

> dfdlx:rightShift not supported ...

Same goes for left shift.